src/net/java/sip/communicator/impl/protocol/jabber/extensions/jibri/JibriStatusPacketExt.java

```diff
@@ -64,57 +63,48 @@ static public void registerExtensionProvider()
         );
     }
 
-    public Status getStatus()
+    private <T extends ExtensionElement> T getFirstChildExtensionOrNull(Class<T> type)
```

Comment:
Should this go to AbstractPacketExtension? Seems generic and potentially useful

Reply:
sure, maybe i should just call it `getChildExtension` (and keep this same behavior) to match the `setChildExtension` i added?

Reply:
Since this is Java 8+, instead of getFirstChildExtensionOrNull name this `Optional<T> getFirstChildExtension`. This is the _perfect_ use case for `Optional`s.